docs/config/development/feature-from-scratch/index.rst

```diff
@@ -0,0 +1,385 @@
+
+                1. Add the ``structure-sponge-loader`` addon to the pack manifest, using versions ``1.+``
+
+                2. Save your structure using `WorldEdit <https://worldedit.enginehub.org/en/latest/usage/clipboard/>`__.
+
+                3. Add the ``.schem`` file to your pack.
+
+                ``oak_tree.schem`` will be used for this guide.
+
+
+3. Create your feature config
+-----------------------------
+
+.. card::
+
+    Add the ``config-feature`` addon to the pack manifest, using versions ``1.+``.
+
+    This addon will allow us to create feature config files.
+
+    .. code-block:: yaml
+        :caption: pack.yml
+        :linenos:
+        :emphasize-lines: 7
+
+        id: YOUR_PACK_ID
+
+        version: 0.1.0
+
+        addons:
+          ...
+          config-feature: "1.+"
+
+    :ref:`Create a blank config file <create-config-file>` and open it your editor.
+
+    Set the :ref:`config type <config-types>` via the ``type``
+    parameter, and config ``id`` as shown below. ``oak_trees.yml`` will be used for this guide.
+
+    .. code-block:: yaml
+        :caption: oak_trees.yml
+        :linenos:
+
+        id: OAK_TREES
+        type: FEATURE
+
+4. Create the feature distributor
+---------------------------------
+
+.. card::
+
+    Distributors determine the x-axis and z-axis placement of a feature in the world.
+
+    Add the ``config-distributors`` addon to the pack manifest, using versions ``1.+``.
+
+    This addon will allow us to create distributors within feature config files.
+
+    .. code-block:: yaml
+        :caption: pack.yml
+        :linenos:
+        :emphasize-lines: 7
+
+        id: YOUR_PACK_ID
+
+        version: 0.1.0
+
+        addons:
+          ...
+          config-distributors: "1.+"
+
+    Configure the ``oak_trees.yml`` config to utilize ``PADDED_GRID`` distributor type as shown below.
+
+    .. code-block:: yaml
+        :caption: oak_trees.yml
+        :linenos:
+        :emphasize-lines: 4-8
+
+        id: OAK_TREES
+        type: FEATURE
+
+        distributor:
+          type: PADDED_GRID
+          width: 12
+          padding: 4
+          salt: 5864
+
+    The ``PADDED_GRID`` distributor type utilizes cells in a grid with
+    the feature placed within each cell with padding between each cell
+    to ensure that features don't generate too close to one another.
+
+    ``PADDED_GRID`` utilizes the parameters ``width``, ``padding``, and ``salt``.
+
+    * ``Width`` - Determines the size of each cell that will contain your feature
+    * ``Padding`` - Determines the gap between each cell
+    * ``Salt`` - Offsets the results of the distributor to prevent overlap
+
+5. Create the feature locator
+-----------------------------
+
+.. card::
+
+    Locators determine the y-axis placement of a feature in the world.
+
+    Add the ``config-locators`` addon to the pack manifest, using versions ``1.+``.
+
+    This addon will allow us to create locators within feature config files.
```

Comment:
Same as distributor opening, would be more correct to say the addon provides a set of locators to use